binderhub/base.py

```diff
@@ -216,6 +216,13 @@ class VersionHandler(BaseHandler):
     # (e.g. mybinder.org federation when blocking cloud datacenters)
     skip_check_request_ip = True
 
+    def set_default_headers():
```

Comment:
missing self:

```suggestion
    def set_default_headers(self):
```

Reply:
:face_holding_back_tears: I can't even see how one would accidentally do that! :laughing: 

Thanks.